### ISSUE TITLE
Portability

### ISSUE THREAD

**Issue by videah** [josefnpat/LoverNet#2]
If bitser is not in the same folder as main.lua, LoverNet can't find it.

**Comment by josefnpat** [comment]
Unless I'm mistaken, windows does not use soft links correctly, which is what I am using excessively.

```
seppi@seppi7:~/repos/lovernet/client♠ ls -lah
total 16K
drwxr-xr-x 2 seppi seppi 4.0K Apr  3 14:50 .
drwxr-xr-x 5 seppi seppi 4.0K Apr  1 20:09 ..
lrwxrwxrwx 1 seppi seppi   13 Apr  3 14:50 bitser.lua -> ../bitser.lua
-rw-r--r-- 1 seppi seppi  143 Apr  1 17:37 conf.lua
lrwxrwxrwx 1 seppi seppi   13 Mar 28 18:30 define.lua -> ../define.lua
lrwxrwxrwx 1 seppi seppi   15 Mar 28 18:27 lovernet.lua -> ../lovernet.lua
-rw-r--r-- 1 seppi seppi 3.2K Apr  1 20:23 main.lua
```

Obviously the workaround is to replace the soft links with the files themselves, but then from a development perspective you lose the purpose of having the softlinks: keeping the client and server up to date.

In my personal projects, everything is developed on Linux (with deployment being `zip -r ../out.love *`), but obv I would love support from other developers - which means I should probably work in a portable environment (which is what you're suggesting I think).

I have an idea to resolve this issue to avoid softlinks: I would combine the client and server example into one example that a user could either choose to make a client or a server - further showing how extendible the software is.

Any other ideas?

**Comment by josefnpat** [comment]
d16b519faad17498e6fbdf44aac070c65879b453

**Comment by josefnpat** [comment]
Fixed this with a6ddc96571a187cfc9a35684d8ea49700c3ea51d 

@VideahGams please tell me if this works on windows cleanly!
